packages/components/src/components/tabs/tabs.jsx

```diff
@@ -119,47 +129,56 @@ const Tabs = ({
                         'dc-tabs__list--center': center,
                         'dc-tabs__list--header-fit-content': header_fit_content,
                         [`dc-tabs__list--${className}`]: className,
+                        'dc-tabs__list--overflow-hidden': is_scrollable,
```

Comment:
It would make more sense to add a more explicit prop name, e.g. `is_overflow_hidden`. It's perfectly ok to pass two props `is_overflow_hidden` and `is_scrollable` as their names suggest their functions are different.